src/Lean/Meta/CompletionName.lean

```diff
@@ -0,0 +1,41 @@
+/-
```

Comment:
I feel like this file name doesn't accurately describe what's in it. If this is to live in Lean.Meta, it should have a more general name than "completion" or "blacklist w.r.t. completion". Unfortunately, I can't think of anything better than something like "UserFacingName", either.